lib/debug/tracer.rb

```diff
@@ -168,11 +169,11 @@ def setup
               end
 
             when :rest
-              next unless name
+              next unless name && name.to_s != "*"
```

Comment:
FYI: You only need to use `:'*'`

Reply:
ah, right 🤦 
I've opened https://github.com/ruby/debug/pull/212 for this.